restlight-ext/restlight-ext-validator-starter/src/main/java/esa/restlight/ext/validator/starter/SpringMvcValidationExceptionHandler.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 OPPO ESA Stack Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package esa.restlight.ext.validator.starter;
+
+import esa.httpserver.core.AsyncRequest;
+import esa.httpserver.core.AsyncResponse;
+import esa.restlight.ext.validator.ValidationExceptionHandler;
+import esa.restlight.springmvc.util.ResponseStatusUtils;
+import io.netty.handler.codec.http.HttpResponseStatus;
+
+public class SpringMvcValidationExceptionHandler extends ValidationExceptionHandler {
```

Comment:
This class should not be placed in restlight-ext-validator

Reply:
done!